2006-02-04  Robert Millan  <robertmh@gnu.org>

	* dlls/ntdll/file.c: On kernel of FreeBSD, include <sys/mount.h>
	regardless of where statfs was found (since we need the flag macros).

Index: dlls/ntdll/file.c
===================================================================
RCS file: /home/wine/wine/dlls/ntdll/file.c,v
retrieving revision 1.117
diff -u -r1.117 file.c
--- dlls/ntdll/file.c	27 Jan 2006 15:22:02 -0000	1.117
+++ dlls/ntdll/file.c	4 Feb 2006 19:34:30 -0000
@@ -57,6 +57,10 @@
 # endif
 #endif
 
+#if defined(__FreeBSD_kernel__) && (!defined(MNT_RDONLY) || !defined(MNT_LOCAL))
+# include <sys/mount.h>
+#endif
+
 #ifdef HAVE_IOKIT_IOKITLIB_H
 # include <IOKit/IOKitLib.h>
 # include <CoreFoundation/CFNumber.h> /* for kCFBooleanTrue, kCFBooleanFalse */
